mllib/src/test/scala/org/apache/spark/mllib/linalg/distributed/IndexedRowMatrixSuite.scala

```diff
@@ -89,11 +89,19 @@ class IndexedRowMatrixSuite extends SparkFunSuite with MLlibTestSparkContext {
 
   test("toBlockMatrix") {
     val idxRowMat = new IndexedRowMatrix(indexedRows)
+
+    // Tests when n % colsPerBlock != 0
```

Comment:
This only tests a `IndexedRowMatrix` consisted of `DenseVector`. We should add another test to test the cases of `SparseVector` and the mix of them.